src/routes/login/Login.tsx

```diff
@@ -36,14 +33,20 @@ const Login = (): JSX.Element => {
 
     useEffect(() => {
         if (forceLogout && authenticated) {
-            dispatch(logOut()).then(() =>
-                setLocation(encodeURI(`${PATHS.LOGIN}?next=${nextLocation}`))
-            );
+            const logOut = async () => {
+                const logOut = await fetchLogout();
+                if (logOut !== null) {
```

Comment:
I don't think there was a check on the success of the logout action, the setLocation was always performed. I think you can remove this check and remove the return value in fetchLogout.